CHANGELOG

```diff
@@ -132,6 +132,10 @@ Miscellaneous
   assembly routines (although the latter is still possible, by leaving the new
   virtual method as its default base implementation of returning `NULL`).
 
+- Updated benchmarks to work with newest Google Benchmarks classes.
+
+- Renamed the 'MFEM_ENABLE_GOOGLE_BENCHMARKS' option to 'MFEM_ENABLE_BENCHMARKS'.
```

Comment:
```suggestion
- Updated the benchmarks (in tests/benchmarks) to work with the latest Google
  Benchmarks classes (version 1.9.1). Renamed the MFEM_ENABLE_GOOGLE_BENCHMARKS
  CMake option to just MFEM_ENABLE_BENCHMARKS.
```

Reply:
done in https://github.com/mfem/mfem/pull/4723/commits/a0723083c65ff1dc1712e79e1683c40268658055